README.md

```diff
@@ -37,91 +34,83 @@ If you'd like to use the accelerator-specific features of 🤗 Optimum, you can
 
 ```bash
 python -m pip install git+https://github.com/huggingface/optimum.git#egg=optimum[onnxruntime]
 ```
 
+## Accelerated Inference
+
+🤗 Optimum provides multiple tools to export and run optimized models on various ecosystems: 
+
+- ONNX / ONNX Runtime
+- TensorFlow Lite
+- OpenVINO
 
-## Optimizing models towards inference
+While everything can be done in a programmatic way for a more cutomized experience, we provide a command-line interface to perform common tasks easily. 
 
-Along with supporting dedicated AI hardware for training, Optimum also provides inference optimizations towards various frameworks and
-platforms.
+### ONNX + ONNX Runtime
 
-Optimum enables the usage of popular compression techniques such as quantization and pruning by supporting [ONNX Runtime](https://onnxruntime.ai/docs/) along with Intel [Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) and OpenVINO [NNCF](https://docs.openvino.ai/latest/tmo_introduction.html).
+It is possible to export 🤗 Transformers models to the [ONNX](https://onnx.ai/) format and perform graph optimization as well as quantization easily:
 
-| Features                           | ONNX Runtime          |     Neural Compressor   |         OpenVINO        |
-|:----------------------------------:|:---------------------:|:-----------------------:|:-----------------------:|
-| Post-training Dynamic Quantization |  :heavy_check_mark:   |    :heavy_check_mark:   |          N/A            |
-| Post-training Static Quantization  |  :heavy_check_mark:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
-| Quantization Aware Training (QAT)  |  Stay tuned! :star:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
-| Pruning                            |        N/A            |    :heavy_check_mark:   |    :heavy_check_mark:   |
+```plain
+optimum-cli export onnx -m deepset/roberta-base-squad2 --optimize O2 roberta_base_qa
+```
 
-## Quick tour
+The model can then be quantized using `onnxruntime`:
 
-Check out the examples below to see how 🤗 Optimum can be used to train and run inference on various hardware accelerators.
+```bash
+optimum-cli onnxruntime quantize --avx512 --onnx_model roberta_base_qa
+```
 
-## Accelerated inference
+These commands will export `deepset/roberta-base-squad2` perform [O2 graph optimization](https://huggingface.co/docs/optimum/onnxruntime/usage_guides/optimization#optimization-configuration) on the exported model, and finally quantize it.
 
-#### ONNX Runtime
+For more information on the ONNX export, please check the [documentation](https://huggingface.co/docs/optimum/exporters/onnx/usage_guides/export_a_model).
 
-To accelerate inference with ONNX Runtime, 🤗 Optimum uses _configuration objects_ to define parameters for graph optimization and quantization. These objects are then used to instantiate dedicated _optimizers_ and _quantizers_.
+#### Run the exported model using ONNX Runtime
```

Comment:
I can work on adding a table/plot highlighting why one would want to run inference with ORT